### PR TITLE
first commit / clock

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
- [DEMO LINK](https://NikitaKarbazin.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      height: 100%;
+      transform-origin: bottom;
+
+      &--minute {
+        position: absolute;
+        width: $width-hand;
+        height: 100px;
+        border-radius: $borderRadiusHand;
+        background-color: $bgMin;
+        left: calc(50% - 2px);
```
Comment (Kaunin1) [inline src/styles/main.scss:54]: - Could you please  change the name of variable to the one with more or less clear meaning.
 - Reagrding 2px - kindly look through above comment

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        animation: sec 3600s steps(60, end) infinite;
+      }
+
+      &--second {
+        position: absolute;
+        width: $width-hand;
+        height: 150px;
+        border-radius: $borderRadiusHand;
+        background-color: $bgSec;
+        left: calc(50% - 2px);
```
Comment (Kaunin1) [inline src/styles/main.scss:65]: Same ? kindly adjust  name of variable  and representation off 2px adjustment of position.

File: src/styles/main.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    box-shadow: 27px 14px 15px -10px rgba(16, 205, 193, 0.69);
+
+    &::after {
+      content: "";
+      position: absolute;
+      @include circle($size-circle);
+      background-color: $bgMin;
+      left: calc(50% - 9px);
+      top: calc(50% - 9px);
+    }
```
[the remark this answers is not in the document]
Comment (NikitaKarbazin) [reply src/styles/main.scss:41]: fixed

File: src/styles/main.scss (hunk 1 above)
Comment (NikitaKarbazin) [reply src/styles/main.scss:54]: fixed

File: src/styles/main.scss (hunk 2 above)
Comment (NikitaKarbazin) [reply src/styles/main.scss:65]: fixed

**Review by Kaunin1 — APPROVED** [review]

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**